src/coreclr/src/vm/dllimport.cpp

```diff
@@ -51,6 +51,9 @@
 #include "clr/fs/path.h"
 using namespace clr::fs;
 
+// Specifies whether coreclr is embedded or standalone
+extern bool g_coreclr_embedded;
```

Comment:
Where are we setting this variable? I cannot see it in this PR.

Reply:
the variable was introduced when we did coreclr embedding (previous PR of this series). It relies on coreclr mergeable knowing this fact at compile time.

Reply:
https://github.com/dotnet/runtime/blob/371ae3c0e8fcc064eae7a61d91e0b04bf6a5738e/src/coreclr/src/vm/ceemain.cpp#L242

Reply:
Ah, thanks for the details, I've missed this one, I somehow thought we had just the g_running_in_exe.

Reply:
`g_running_in_exe` is the PAL mirror of `g_coreclr_embedded`, since it can't see `g_coreclr_embedded` directly.